Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -1406,15 +1385,12 @@ std::vector<PAIR_I_I> EmbeddedFrag::findCollisions(const double *dmat,
     d_eatom.second.d_density = 0.0;
   }
 
-  auto tempi = d_eatoms.begin();
-  ++tempi;
   auto colThres2 = COLLISION_THRES * COLLISION_THRES;
   // if we a re dealing with non carbon atoms we will increase the collision
   // threshold. This is because only hetero atoms are typically drawn in a
   // depiction.
   double atomTypeFactor1, atomTypeFactor2;
-  for (auto efi = tempi; efi != d_eatoms.end(); ++efi) {
-    auto pti = efi->second.loc;
+  for (auto efi = d_eatoms.begin(); efi != d_eatoms.end(); ++efi) {
```

Comment:
This is different from how it was previously: now the iterator starts from `d_eatoms.begin()`, previously it was starting from `d_eatoms.begin() + 1`.

Reply:
yeah, introducing that extra variable makes the code a bit trickier to read and was unlikely to have any remotely measurable impact on performance. The final behavior this way is the same.